### PR TITLE
Resolve 4# Added new footer feature

### PR DESCRIPTION
**Pull request by abalanag**
Added a new feature with the following modifications:
* Created the project footer on the footer.html file
* Created the link to the Bootstrap styles on the common html file

### CODE REVIEW

File: src/main/resources/templates/common.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+          integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script th:fragment="bootstrap-js"
+            src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+            crossorigin="anonymous"></script>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (danutchindris) [inline src/main/resources/templates/common.html]: Please always add an empty line to each file you commit to the Git repository.

File: src/main/resources/templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    </div>
+
+
+    <!-- Copyright -->
+    <div class="text-center text-dark p-3"> © 2020 Copyright:
+        <a class="text-dark" href="//localhost">Alin AirLines</a>
+    </div>
+</footer>
+</body>
+</html>
```
Comment (danutchindris) [inline src/main/resources/templates/footer.html]: Same as above.

File: src/main/resources/templates/footer.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Footer</title>
+
+</head>
+<body>
+<footer class="text-center text-white" style="background-color: rgba(0, 0, 0, 0.2);" th:fragment="footer">
```
Comment (danutchindris) [inline src/main/resources/templates/footer.html]: Adding a `style` attribute is discouraged. This should be used only as a last resort.
We should use cascading style sheets whenever possible.

File: src/main/resources/templates/footer.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                </svg>
+            </a>
+        </section>
+    </div>
+
+
+    <!-- Copyright -->
+    <div class="text-center text-dark p-3"> © 2020 Copyright:
```
Comment (danutchindris) [inline src/main/resources/templates/footer.html]: Use `&copy;` instead.

Comment (danutchindris) [reply src/main/resources/templates/footer.html]: It's 2021 already 😁

File: src/main/resources/templates/footer.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        <a class="text-dark" href="//localhost">Alin AirLines</a>
```
Comment (danutchindris) [inline src/main/resources/templates/footer.html]: Write a proper URL for this anchor element, using the Thymeleaf specific attribute.

File: src/main/resources/templates/footer.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    <title>Footer</title>
+
+</head>
+<body>
+<footer class="text-center text-white" style="background-color: rgba(0, 0, 0, 0.2);" th:fragment="footer">
+    <div class="container pt-4">
+        <section class="mb-4">
+            <!-- Facebook -->
+            <a href="https://google.com" type="button" class="btn btn-secondary">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
```
Comment (danutchindris) [inline src/main/resources/templates/footer.html]: Is there no other solution than writing these `svg` elements inside the markup?
Maybe you can have actual SVG files as static resources and reference them in the markup.

**Review by danutchindris — CHANGES_REQUESTED** [review]
Please see the review comments and perform the necessary changes.
Thanks!

File: src/main/resources/templates/common.html (hunk 1 above)
Comment (abalanag) [reply src/main/resources/templates/common.html]: Empty line added

File: src/main/resources/templates/footer.html (hunk 2 above)
Comment (abalanag) [reply src/main/resources/templates/footer.html]: Empty line added

File: src/main/resources/templates/footer.html (hunk 3 above)
Comment (abalanag) [reply src/main/resources/templates/footer.html]: Created style.css file in which was added the background color.

File: src/main/resources/templates/footer.html (hunk 4 above)
Comment (abalanag) [reply src/main/resources/templates/footer.html]: Traveled back to the year 2021, line updated.

File: src/main/resources/templates/footer.html (hunk 5 above)
Comment (abalanag) [reply src/main/resources/templates/footer.html]: I hope this will make the trick 😄

File: src/main/resources/templates/footer.html (hunk 6 above)
Comment (abalanag) [reply src/main/resources/templates/footer.html]: Created SVG files for all the images and updated lines.

**Review by danutchindris — APPROVED** [review]
OK!

**Pull request merged**